web/src/components/DandiFooter.vue

```diff
@@ -15,7 +15,7 @@
       </cookie-law>
       <v-row>
         <v-col offset="2">
-          &copy; 2019 - 2024 The DANDI Team<br>
+          &copy; 2019 - 2025 The DANDI Team<br>
```

Comment:
Hi @jjnesbitt, could we use the `Date` object (`new Date().getFullYear()`)?  I presume we would need to declare the constant elsewhere and inject it in the footer.  [Stackoverflow post](https://stackoverflow.com/questions/6002254/get-the-current-year-in-javascript).

Reply:
We could do that, sure.

Reply:
I think automation is overkill for this. The yearly manual update should be just fine.